docs/inso-cli/install.md

```diff
@@ -5,18 +5,33 @@ category: "Inso CLI"
 category-url: inso-cli
 ---
 
+## Install single executable
+
+Inso CLI can be downloaded and run as a single executable on MacOS, Windows or Linux. Download the release artifacts from [GitHub Releases](https://github.com/Kong/insomnia/releases/tag/lib%402.4.0).
```

Comment:
The link doesn't exist right now but will once the release artifacts have been created.

I've added INS-1087 to make sure we don't have to link to a specific version in the future.